<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphView</class>
 <widget class="QWidget" name="GraphView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="loadingLabel">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>631</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Loading graph... Please, wait</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>450</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_6">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>460</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_7">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_8">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_9">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_10">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_11">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_12">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_13">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_14">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_15">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trainObject_16">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px; background-color: lightgreen; color: black; font-weight: 600; text-align: center; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="town_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="town_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>470</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="town_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="town_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>470</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="market_1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="market_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="market_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>470</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="market_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>470</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="storage_1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="storage_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="storage_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>260</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="storage_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
